examples/create-react-app-material-ui/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "create-react-app-material-ui",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "material-ui": "^1.0.0-alpha.22",
```

Comment:
I haven't used next.js yet. I can add it after getting everything else correctly. What do you think?